backend/src/main/java/ai/verta/modeldb/dataset/DatasetDAORdbImpl.java

```diff
@@ -323,6 +325,17 @@ public void deleteDatasetVersionsByDatasetIDs(Session session, List<String> data
     List<DatasetVersionEntity> datasetVersionEntities = query.list();
     for (DatasetVersionEntity datasetVersionEntity : datasetVersionEntities) {
       session.delete(datasetVersionEntity);
+
+      String ownerRoleBindingName =
```

Comment:
so this was a bug in our system?
In future please extract bug fixes out so that they can be merged in sooner

Reply:
No this is not a bug. this functionality is new where we delete the entity owner. in this case (when deleting Dataset we are deleting DatasetOwner ROle binding.